File: 백준/Bronze/5532. 방학 숙제/방학 숙제.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        
        int L = sc.nextInt();
        int A = sc.nextInt();
        int B = sc.nextInt();
        int C = sc.nextInt();
        int D = sc.nextInt();
        
        int kor = 0;
		int math = 0;
        
        if (A % C == 0) {
			kor = L - (A / C);
		} else {
			kor = L - (A / C + 1);
		}

		if (B % D == 0) {
			math = L - (B / D);
		} else {
			math = L - (B / D + 1);
		}

		System.out.println(Math.min(kor, math));
    }
}

File: 백준/Bronze/5532. 방학 숙제/방학 숙제.cs
using System;

class Q5532
{
    static void Main(string[] args)
    {
        int L = int.Parse(Console.ReadLine());
        int A = int.Parse(Console.ReadLine());
        int B = int.Parse(Console.ReadLine());
        int C = int.Parse(Console.ReadLine());
        int D = int.Parse(Console.ReadLine());
        
        int kor = 0;
		int math = 0;
        
        if (A % C == 0)
        {
			kor = L - (A / C);
		}
        else
        {
			kor = L - (A / C + 1);
		}

		if (B % D == 0)
        {
			math = L - (B / D);
		}
        else
        {
			math = L - (B / D + 1);
		}

		Console.WriteLine(Math.Min(kor, math));
    }
}